packages/nodejs/src/client.ts

```diff
@@ -73,7 +78,14 @@ export class BaseClient implements Client {
 
     this.instrumentation = new Instrumentation(this.tracer(), this.metrics())
 
-    initCorePlugins(this.instrumentation, { ignoreInstrumentation })
+    initCorePlugins(this.instrumentation, {
+      instrumentationConfig: {
+        http: instrumentHttp || true,
+        https: instrumentHttps || true,
+        pg: instrumentPg || true,
+        redis: instrumentRedis || true
```

Comment:
P.S. to the comment above: when reading the options from the `Configuration` object, their default `true` value from the `Configuration` defaults will apply, so it won't be necessary to add `|| true` to each of them.

Reply:
Good point. Looking at this with fresh eyes you can't disable it through the client. I run into the issue that if I remove `|| true` I get this error:

```
src/client.ts:82:9 - error TS2322: Type 'boolean | undefined' is not assignable to type 'boolean'.
  Type 'undefined' is not assignable to type 'boolean'.

82         http: instrumentHttp,
           ~~~~
```

I'll take a better look

Reply:
Ah, yes, the types on the config. Since `this.config.data` is a `Partial<AppsignalOptions>`, every key in it could be undefined -- even if we know it won't be, because it has a default value. I wonder if it really needs to be declared as a partial. If every configuration option has a default, then it wouldn't need to be a partial.

Anyway, feel free to ignore my original comment. No need to mess with the types at this point.